sklearn/base.py

```diff
@@ -322,7 +320,10 @@ def __setstate__(self, state):
                 warnings.warn(
                     "Trying to unpickle estimator {0} from version {1} when "
                     "using version {2}. This might lead to breaking code or "
-                    "invalid results. Use at your own risk.".format(
+                    "invalid results. Use at your own risk. "
+                    "For more info please refer to:\n\n"
```

Comment:
Here as well:

```suggestion
                    "For more info please refer to:\n"
```

Reply:
I have made the changes you suggested!

Yes there are unrelated changes because I'm had installed other black version, not the suggested in the Scikit-Learn contribution guide. Now everything is fine!